Re-import css/css-images/gradient

roberto-apple opened the pull request:
#### 88e269109acac005c4ca59efe68bed066053917e
<pre>
Re-import css/css-images/gradient
<a href="https://bugs.webkit.org/show_bug.cgi?id=286446">https://bugs.webkit.org/show_bug.cgi?id=286446</a>
<a href="https://rdar.apple.com/143529813">rdar://143529813</a>

Reviewed by Tim Nguyen.

Re-import wpt css/css-images/gradient tests
Upstream commit: web-platform-tests/wpt@5753f6c

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/color-stops-parsing-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/color-stops-parsing.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-decreasing-hue-hsl-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-decreasing-hue-hsl-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-decreasing-hue-hsl.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-decreasing-hue-lch-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-decreasing-hue-lch-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-decreasing-hue-lch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-hue-direction-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-hue-direction-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-hue-direction.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-increasing-hue-hsl-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-increasing-hue-hsl-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-increasing-hue-hsl.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-increasing-hue-lch-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-increasing-hue-lch-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-increasing-hue-lch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-006-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-006-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-007-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-007-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-008-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-008-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-009-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-009-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-009.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-010-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-010-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-010.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-011-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-011-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-011.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-012-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-012-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-012.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-006-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-006-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-007-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-007-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-008-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-008-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-009-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-009-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-009.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-010-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-010-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-010.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-011-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-011-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-011.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-012-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-012-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-012.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-006-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-007-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-008-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-hsl-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-hsl-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-hsl-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-hsl-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-hsl-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-hsl.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-oklch-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-oklch-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-oklch.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/289372@main">https://commits.webkit.org/289372@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/bab719e41816bf48e8e4e4202822097941121dbf

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/86654 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/6161 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/40986 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/91498 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/37386 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/6429 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/14219 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/67012 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/24799 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/89657 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/4885 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/78457 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/47335 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/4684 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/32788 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/36504 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/75135 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/33673 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/93379 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/13807 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/10009 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/75811 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/14008 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/74279 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/74998 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/19297 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/17697 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/6592 "The change is no longer eligible for processing.") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/13479 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/13830 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/13568 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/17012 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/15352 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
LGTM, please fix the bug link and optionally the radar link in the commit message before landing

roberto-apple committed:
Re-import css/css-images/gradient
https://bugs.webkit.org/show_bug.cgi?id=286446
rdar://143529813

Reviewed by Tim Nguyen.

Re-import wpt css/css-images/gradient tests
Upstream commit: web-platform-tests/wpt@5753f6c

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/color-stops-parsing-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/color-stops-parsing.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-decreasing-hue-hsl-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-decreasing-hue-hsl-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-decreasing-hue-hsl.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-decreasing-hue-lch-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-decreasing-hue-lch-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-decreasing-hue-lch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-hue-direction-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-hue-direction-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-hue-direction.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-increasing-hue-hsl-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-increasing-hue-hsl-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-increasing-hue-hsl.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-increasing-hue-lch-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-increasing-hue-lch-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-increasing-hue-lch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-006-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-006-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-007-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-007-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-008-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-008-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-009-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-009-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-009.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-010-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-010-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-010.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-011-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-011-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-011.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-012-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-012-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-hsl-012.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-004-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-005-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-006-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-006-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-007-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-007-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-008-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-008-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-009-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-009-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-009.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-010-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-010-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-010.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-011-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-011-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-011.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-012-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-012-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-longer-hue-lch-012.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-005-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-005.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-006-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-006.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-007-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-007.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-008-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-008.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-hsl-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-hsl-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-hsl-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-hsl-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-hsl-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-hsl.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-oklch-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-oklch-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/gradient-single-stop-longer-hue-oklch.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-images/gradient/w3c-import.log:

Canonical link: https://commits.webkit.org/289372@main
